api-reference/beta/resources/security_overview.md

```diff
@@ -0,0 +1,46 @@
+# Use Microsoft Graph to integrate with the Intelligent Security Graph
+
+ > **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+The Microsoft Intelligent Security Graph is a unified platform for combatting cyberthreats. It powers real-time threat protection for Microsoft products and services and supports an ecosystem of integrated solutions.

```

Comment:
Heads-up: checking with internal folks for guidance on "the" before "Microsoft Intelligent Security Graph".